src/tests/ov_helpers/ov_lpt_models/src/mat_mul.cpp

```diff
@@ -49,36 +50,94 @@ std::shared_ptr<ov::Model> MatMulFunction::getOriginal(
+                { 0.f }, { 255.f / 4.f }, { 0.f }, { 255.f / 4.f });
     fakeQuantizeOnAcitvations->set_friendly_name("fakeQuantizeOnAcitvations");
 
-    auto weightsConst = std::make_shared<ov::op::v0::Constant>(
-        precision,
-        inputShape2.to_shape(),
-        std::vector<float>({ 1.f }));
-    const auto fakeQuantizeOnWeights = ov::test::utils::make_fake_quantize(
-        weightsConst, precision, 256ul, { 1ul, 1ul },
-        { -128.f / 8.f }, { 127.f / 8.f }, { -128.f / 8.f }, { 127.f / 8.f });
-    fakeQuantizeOnWeights->set_friendly_name("fakeQuantizeOnWeights");
+    const size_t channel = inputShape2[inputShape2.size() - 2].get_length();
+
+    // fq
+    std::shared_ptr<Node> parentOnWeights;
+    if (fq) {
+        auto weightsConst = ov::test::utils::make_constant(precision, inputShape2.to_shape());
+        parentOnWeights = perChannelWeightsDequantization ?
+                          ov::test::utils::make_fake_quantize(
+                                  weightsConst, precision, 256ul,
+                                  Shape{channel, 1},
+                                  generate_dequantization_values(Shape{channel, 1}, 256ul, true),
+                                  generate_dequantization_values(Shape{channel, 1}, 256ul, false),
+                                  generate_dequantization_values(Shape{channel, 1}, 256ul, true),
+                                  generate_dequantization_values(Shape{channel, 1}, 256ul, false)) :
+                          ov::test::utils::make_fake_quantize(
+                                  weightsConst, precision, 256ul, {1ul, 1ul},
+                                  {-128.f / 8.f}, {127.f / 8.f}, {-128.f / 8.f}, {127.f / 8.f});
+    } else {
+        Shape shape = inputShape2.to_shape();
+        if (transpose2) {
+            shape[shape.size() - 1ull] = 1;
+        } else {
+            shape[shape.size() - 2ull] = 1;
+        }
+        auto weightsConst = ov::test::utils::make_constant(signedOnWeights ? element::i8 : element::u8, inputShape2.to_shape(), {});
+        const auto convert = std::make_shared<opset1::Convert>(weightsConst, precision);
+
+        const auto multiplyConst = ov::test::utils::make_constant(precision, shape);
+        parentOnWeights = std::make_shared<opset1::Multiply>(convert, multiplyConst);
+    }
 
-    const std::shared_ptr<ov::opset1::MatMul> fullyConnected = std::make_shared<ov::opset1::MatMul>(
+    parentOnWeights->set_friendly_name("fakeQuantizeOnWeights");
+
+    std::shared_ptr<Node> parent = std::make_shared<ov::opset1::MatMul>(
         fakeQuantizeOnAcitvations->output(0),
-        fakeQuantizeOnWeights->output(0),
+        parentOnWeights->output(0),
         transpose1,
         transpose2);
-    fullyConnected->set_friendly_name("fullyConnected");
+    parent->set_friendly_name("fullyConnected");
+
+    if (bias) {
+        auto bias = ov::test::utils::make_constant(precision, parent->get_output_shape(0));
```

Comment:
1. Please use `ov::PartialShape` in builders, since they can be used for dynamic shapes
2. It's better to construct the shape of bias in the following way:
```
ov::Shape bias_shape(parent->get_output_partial_shape(0).size(), 1);
bias_shape.back() = parent->get_output_partial_shape(0).back().get_length();
```

Reply:
Fixed